test/e2e/operator_test.go

```diff
@@ -175,69 +175,6 @@ func TestClusterIngressControllerCreateDelete(t *testing.T) {
 	}
 }
 
-func TestRouterServiceInternalEndpoints(t *testing.T) {
```

Comment:
Curious to know why this test is being removed.

Reply:
IMO the test isn't buying us much other than asserting the service exists, which is covered implicitly elsewhere. It did check the endpoints, but that's such a specific test you could cover with a unit (e.g. do the selectors match) and there's origin e2e coverage for metrics which actually uses the service ([currently broken](
https://bugzilla.redhat.com/show_bug.cgi?id=1687640)).

Reply:
Spoke offline and decided to do a followup which introduces a new test variant which exercises the metrics endpoint through the service (which will both implicitly test the endpoints and also the actual functional area around the internal service, which is enabling metrics integration)